README.md

```diff
@@ -1,76 +1,106 @@
 triton-elk
 ==========
 
-[Elasticsearch-Logstash-Kibana (ELK)](https://www.elastic.co/products) stack designed for container-native deployment on Joyent's Triton platform.
+*[Autopilot pattern](http://autopilotpattern.io/) implementation of ELK*
 
-### Architecture
+This repo is a demonstration of the [Elasticsearch-Logstash-Kibana (ELK)](https://www.elastic.co/products) stack, designed to be self-operating according to the [autopilot pattern](http://autopilotpattern.io/). This application also demonstrates Triton support of Docker log drivers.
+
+The components of this stack are:
+
+- [Elasticsearch](https://www.elastic.co/products/elasticsearch), to store and search our logs.
+- [Logstash](https://www.elastic.co/products/logstash), to ingest logs from our containers and write them to Elasticsearch.
+- [Kibana](https://www.elastic.co/products/kibana), as a web UI to search the logs we store in Elasticsearch.
+- [Consul](https://www.consul.io/), acting a service catalog to support discovery
+- [Containerbuddy](http://containerbuddy.io), to help with service discovery and bootstrap orchestration
+- [Triton](https://www.joyent.com/), Joyent's container-native infrastructure platform, including the new [Docker log drivers](https://www.joyent.com/blog/docker-log-drivers) support.
+- Nginx, acting as a source of logs for testing.
 
 ![Diagram of Triton-ELK architecture](./doc/triton-elk.png)
```

Comment:
There's something about this diagram that makes me think that Logstash is a back-end for Nginx. That interpretation isn't easily justified, as the relationship lines clearly do not always indicate "back-end," as the lines to Consul should make clear, it's just that seeing Nginx without a back-end is so unusual.